research-software-directory/RSD-as-a-service
# Global notifications

Changes proposed in this pull request:

* administrators can create global notifications via the admin interface, e.g. to notify visitors about upcoming downtimes

How to test:

* `make start` to build app and generate test data
* login as rsd_admin to be able to access admin section
* navigate to Anouncement section of admin page
* Fill in info in the form and hit save button
* Reload RSD to view notification. It will be shown on every page. After using X button it will be hidden. This flag is only in memory and if you reload RSD the notification will appear again.
* The notification uses `warning` colors defined in the settings (light theme).

## Example admin section
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/621946bb-1a98-4890-ae24-9f76a258ceac)

## Example announcement
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/df374cf2-cb24-463e-bc6c-50e989285bcd)


PR Checklist:
